packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -836,6 +837,10 @@ export const handlers = [
     ];
     return HttpResponse.json(makeResourcePage(objectStorageTypes));
   }),
+  http.get('*/v4/object-storage/endpoints', ({}) => {
```

Comment:
```suggestion
  http.get('*/v4beta/object-storage/endpoints', ({}) => {
```
My understanding is that this is a new endpoint, and new endpoints should be `v4beta` rather than `v4` - is that correct here?

Reply:
As of right now, API has implemented this using `v4`, but I'm waiting for final reply from API/Architect - this could be a small amendment in a follow-up if I don't get an answer quickly

Reply:
No need to prefix this with `v4beta` since the Beta is currently in progress and not integrated with the linode API at all. This will be the endpoint for GA.